.eslintignore

```diff
@@ -17,6 +17,5 @@ doc/
 build/
 demo/
 api/
-**/* *
```

Comment:
How? if i add an error to makefile.dryice.js and run `node_modules/eslint/bin/eslint.js .` it finds that error.
This pattern should match only if the file has a space in name, unless there is a bug in eslint.